test/README.md

```diff
@@ -80,13 +80,13 @@ to recover with:
 
 ```bash
 rm -rf cache
-killall bitcoind
+killall dashd
 ```
 
 Util tests
 ----------
 
-Util tests can be run locally by running `test/util/bitcoin-util-test.py`. 
+Util tests can be run locally by running `test/util/dash-util-test.py`.
```

Comment:
shouldn't touch this line :)

Reply:
https://github.com/dashpay/dash/pull/2912/commits/381efe77390e7b28d91ed043b0c6d4c2abdc314d